### PR TITLE
OrchAgent add more logging

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by stcheng** [comment]
I think this will be too verbose.
I'll add proper logs in side functions.

**Comment by lguohan** [comment]
debug log, i'm ok with that. In production, should only turn on info or notice level.

**Comment by kcudnik** [comment]
I can change it to INFO, since i just needed this for debug.

**Comment by stcheng** [comment]
I've reviewed the code change and I think not all the places here need to add SWSS_LOG_ENTER();.
I've leaved some comments.

File: orchagent/main.cpp [hunk 1]
```diff
@@ -40,6 +40,8 @@ const char *test_profile_get_value (
     _In_ sai_switch_profile_id_t profile_id,
     _In_ const char *variable)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/main.cpp]: short function. no need to add this.

File: orchagent/main.cpp [hunk 2]
```diff
@@ -52,6 +54,8 @@ int test_profile_get_next_value (
     _Out_ const char **variable,
     _Out_ const char **value)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/main.cpp]: short function. no need to add this.

File: orchagent/main.cpp [hunk 3]
```diff
@@ -92,6 +98,8 @@ void initSaiApi()
 
 void initDiagShell()
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/main.cpp]: it goes into while true in another thread. no need to have it here.

File: orchagent/portsorch.cpp [hunk 4]
```diff
@@ -128,6 +130,8 @@ PortsOrch::PortsOrch(DBConnector *db, string tableName) :
 
 bool PortsOrch::getPort(string alias, Port &p)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/portsorch.cpp]: no need here as the function returns bool and if it is false it always prints out error messages.

File: orchagent/routeorch.cpp [hunk 5]
```diff
@@ -203,28 +213,38 @@ bool RouteOrch::removeNextHopEntry(IpAddresses ipAddresses)
 
 int RouteOrch::getNextHopRefCount(IpAddress ipAddress)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/routeorch.cpp]: not needed for these get functions.

File: orchagent/routeorch.cpp [hunk 6]
```diff
@@ -203,28 +213,38 @@ bool RouteOrch::removeNextHopEntry(IpAddresses ipAddresses)
...
     IpAddresses ip_addresses(ipAddress.to_string());
     return getNextHopRefCount(ip_addresses);
 }
 
 int RouteOrch::getNextHopRefCount(IpAddresses ipAddresses)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/routeorch.cpp]: not needed for these get functions.

File: orchagent/routeorch.cpp [hunk 7]
```diff
@@ -203,28 +213,38 @@ bool RouteOrch::removeNextHopEntry(IpAddresses ipAddresses)
...
     return m_syncdNextHops[ipAddresses].ref_count;
 }
 
 NextHopEntry RouteOrch::getNextHopEntry(IpAddress ipAddress)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/routeorch.cpp]: not needed for these get functions.

File: orchagent/routeorch.cpp [hunk 8]
```diff
@@ -203,28 +213,38 @@ bool RouteOrch::removeNextHopEntry(IpAddresses ipAddresses)
...
     IpAddresses ip_addresses(ipAddress.to_string());
     return getNextHopEntry(ip_addresses);
 }
 
 NextHopEntry RouteOrch::getNextHopEntry(IpAddresses ipAddresses)
 {
+    SWSS_LOG_ENTER();
+
```
Comment (stcheng) [inline orchagent/routeorch.cpp]: not needed for these get functions.

**Comment by kcudnik** [comment]
It will be very helpful when debugging, i can change default level to INFO if you want

**Comment by hrachya-mellanox** [comment]
There are enter/exit logs inside our SAI implementation. They are were helpful for debug and diagnostics.

File: orchagent/main.cpp (hunk 1 above)
Comment (lguohan) [inline orchagent/main.cpp]: it's better to know which values sai library is reading.

**Comment by stcheng** [comment]
ack

**Pull request closed without merging**